convert2rhel/pkgmanager/handlers/yum/__init__.py

```diff
@@ -309,7 +309,7 @@ def _process_transaction(self, validate_transaction):
                 id_="FAILED_TO_VALIDATE_TRANSACTION",
                 title="Failed to validate yum transaction.",
                 description="During the yum transaction execution an error occured and convert2rhel could no longer process the transaction.",
-                diagnosis="Transaction processing failed with error %s." % (str(e)),
+                diagnosis="Transaction processing failed with error %s." % str(e),
```

Comment:
Should we print the error here at all? If there are a lot of packages that were in the transaction this list will be huge

WDYT @r0x0d

Reply:
Better to remove, I think.. Lets leave this to the log.

Reply:
Removed